src/Analytics/UAExperimentData.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace PhpAb\Analytics;
+
+use Webmozart\Assert\Assert;
+
+class UAExperimentData
+{
+
+    /**
+     * @var array Of test identifiers and their variant's indexes
+     */
+    private $testsData = [];
+
+    /**
+     * @param string $testIdentifier It will look like "Qp0gahJ3RAO3DJ18b0XoUQ"
+     * @param int $variationIndex
+     * @return \PhpAb\Analytics\UAExperiments
+     * @throws InvalidArgumentException
+     */
+    public function onRegisterParticipation($testIdentifier, $variationIndex)
+    {
+        Assert::string($testIdentifier, 'Test identifier must be astring');
+
+        Assert::integer($variationIndex, 'Variation index must be integer');
+
+        Assert::greaterThan($variationIndex, -1, 'Variation index must be integer >= 0');
```

Comment:
No, 0 is the original variation:

> Then use the chooseVariation method which will return a number that represents the index of the chosen variation, where 0 is the original variation. The index value returned corresponds to the list of variations configured for the experiment.

https://developers.google.com/analytics/solutions/experiments-client-side#implement-original